napari/_vispy/experimental/vispy_tiled_image_layer.py

```diff
@@ -71,13 +75,20 @@ class VispyTiledImageLayer(VispyImageLayer):
     """
 
     def __init__(self, layer: OctreeImage):
+
         # All tiles are stored in a single TileImageVisual.
         visual = TiledImageNode(tile_shape=layer.tile_shape)
 
         # Pass our TiledImageVisual to the base class, it will become our
         # self.node which VispyBaseImage holds.
         super().__init__(layer, visual)
 
+        # Create events after the base class. We have a loaded event that
+        # QtPoll listens to. Because a chunk might be loaded when QtPoll is
+        # totally quiet, no mouse movement, no in-progress loading. We need
+        # to get the polling going so we can load the chunks over time.
+        self.events = EmitterGroup(source=self, auto_connect=True, loaded=None)
```

Comment:
I don't love that we had to do this, but I see why it might be necessary. There had been talk of using more events to do communication from the vispy visual a while back, but we didn't end up needing it. We might go down that path again though, but can cross those bridge when we get to them!

Reply:
The list of things that would go away if were just drawing constantly is growing. But I still like the idea that napari can go 100% quiet. But there's might be some global way to achieve that. Here were are trying to be super precise so napari won't poll or draw one single frame more than it has to.

Maybe there's a way to just draw all the time "if the window is active" and "the user is doing something". Some global kill switch. Things to do seem to be getting complicated just to workaround the issue that we are not always drawing.

Reply:
> Things to do seem to be getting complicated just to workaround the issue that we are not always drawing.

Yeah, definitely something to consider, but agreed I do like that it can "go 100% quiet" too ....